tests/test_sequence.py

```diff
@@ -109,22 +109,6 @@ def test_merge(self):
         assert c.get() == ["a", "b", "1", "c", "x", "y", "z"]
         assert d.merge(c).get() == ["a", "b", "1", "c", "x", "y", "z"]
 
-    def test_name_conflict(self):
-        """
-        Test that we raise an error when resolving conflicting operations between two identically-named sequences.
-        """
-        a = Sequence(id="alice")
-        a.append("a")
-
-        b = Sequence(id="alice")
-        b.append("b")
-
-        with pytest.raises(ValueError):
-            a.merge(b)
-
-        with pytest.raises(ValueError):
-            b.merge(a)
-
```

Comment:
I don't think it's possible anymore for two operations in a sequence to have the same OpId, although I still have the check in the compare code.